src/utils/ethereumMainnetTokenMapping.ts

```diff
@@ -27,10 +26,6 @@ export const getEthereumMainnetTokenLogoURL = (address: string) => {
     uri = 'https://i.imgur.com/zhZo9Bx.jpg'
   }
 
-  if (!uri) {
```

Comment:
same here, some token image will be affect

Reply:
same above, I moved it to outside
[/ethereumMainnetTokenMapping.ts](https://github.com/KyberNetwork/dmm-interface/pull/1034/files/c62b48348836654f8f1e95ef2b786d30d60d047a#diff-e95cc20307d5762e34a7594551a6ca8ede388ae09617edee29fb095c85150a94)